buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ForbiddenApisCliTask.java

```diff
@@ -57,14 +59,6 @@ public void setTargetCompatibility(JavaVersion targetCompatibility) {
         this.targetCompatibility = targetCompatibility;
     }
 
-    public Action<JavaExecSpec> getExecAction() {
```

Comment:
The action can't be an `@Input`, but we need to pick up changes in runtime java version/